tests/models/mra/test_modeling_mra.py

```diff
@@ -314,36 +314,101 @@ class MraModelTest(ModelTesterMixin, PipelineTesterMixin, unittest.TestCase):
     def setUp(self):
         self.model_tester = MraModelTester(self)
         self.config_tester = ConfigTester(self, config_class=MraConfig, hidden_size=37)
-
+    
     def test_config(self):
         self.config_tester.run_common_tests()
 
+    
+    @skipIfRocm
+    def test_determinism(self):
```

Comment:
where did these tests come from ?

Reply:
every function I added like this is coming from an inherited general test class which works on its own and for other models but is showing failure for that specific model in this case it is coming from ModelTesterMixin class from https://github.com/ROCm/transformers/blob/e58394e4f0e9b0ac62e28ef343612dcecdde48ef/tests/test_modeling_common.py#L553